arkfilepicker/src/main/java/space/taran/arkfilepicker/roots/FoldersRepo.kt

```diff
@@ -0,0 +1,51 @@
+package space.taran.arkfilepicker.roots
+
+import android.content.Context
+import com.beust.klaxon.Klaxon
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.nio.file.Path
+import kotlin.io.path.Path
+import kotlin.io.path.notExists
+
+internal class FoldersRepo(private val appCtx: Context) {
```

Comment:
Will it now belong to `ark-filepicker` only or to Navigator too?

Reply:
Navigator has its own `FoldersRepo`, which works faster because it has direct access to the `Room`.
Also `internal` modifier indicates that this class is visible only in the module(`ark-filepicker`).

Reply:
I guess, that will different after implementing https://github.com/ARK-Builders/ark-filepicker/issues/7